src/pages/_app.tsx

```diff
@@ -1,29 +1,30 @@
 import React from "react";
 import { AppProps } from "next/app";
 import Script from "next/script";
-
 import { Refine } from "@pankod/refine";
 import routerProvider from "@pankod/refine-nextjs-router";
-
 import "@pankod/refine/dist/styles.min.css";
-import { DataProvider } from "@pankod/refine-strapi";
-import { API_URL } from "src/constants";
+import { DataProvider } from "src/dataProvider";
+// import { DataProvider } from "@pankod/refine-strapi";
 import GlobalStyle from "../styles/globals";
 import { ThemeProvider } from "styled-components";
 import Header from "../components/common/Header";
 import Footer from "../components/common/Footer";
 import theme from "../styles/theme";
 import { UserDataProvider } from "src/context/UserDataContext";
-import { PostList } from "@components/dashboard/list";
+import {ProjectList} from "@components/dashboard/list";
 import { env } from "src/utils/EnvironmentVariables";
 
+const CustomDashboardPage = () => <div> Custom Dashboard Page </div>;
+
 function MyApp({ Component, pageProps }: AppProps): JSX.Element {
   const dataProvider = DataProvider(env().STRAPI_URL);
```

Comment:
This DataProvider accepts another argument which takes an axios instance. 
I think we can fix the issue of adding "with-credentials" header by adding an axios instance to it rather than changing the library itself. 

try something like
```
const axiosInstance = axios.create();
axiosInstance.defaults.withCredentials = true;
function MyApp({ Component, pageProps }: AppProps): JSX.Element {
  const dataProvider = DataProvider(env().STRAPI_URL, axiosInstance);
  ....
```
NOTE: that you should be using 
`import { DataProvider } from "@pankod/refine-strapi";`

Reply:
Thank you for the detailed comments. That was very helpful. Incorporated suggested changes and pushed the commit to repo